### PR TITLE
Split property component tree into its own directory

### PR DESCRIPTION
**Pull request by jcoyne**
This makes it easier to understand what components are related to properties and which operate at a higher level.

### CODE REVIEW

**Comment by jermnelson** [comment]
Could you provide more context and why you're suggesting this change in the PR's description?

File: .eslintrc.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ module.exports = {
   settings: {
     "import/resolver": {
       "node": {
+        "paths": ['src'],
```
Comment (mjgiarlo) [inline .eslintrc.js]: Can you explain why this branch needs this change?

Comment (mjgiarlo) [reply .eslintrc.js]: Nvm, I think I get it now. It provides a base resolution path so we can get away from the `..` nonsense. Cool! Good change.

File: src/components/editor/property/InputLang.jsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import React, { Component } from 'react'
 import { Typeahead } from 'react-bootstrap-typeahead'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
-import { setLang } from '../../actions/index'
+import { setLang } from '../../../actions/index'
```
Comment (mjgiarlo) [inline src/components/editor/property/InputLang.jsx]: Could the path here not be `'components/actions/index'`? I assumed that's what the changes to `package.json` and `.eslintrc.js` would enable. No?

Comment (mjgiarlo) [reply src/components/editor/property/InputLang.jsx]: Or is it more that you're not sanguine about committing to nuking each and every `'../'` import in the codebase? That'd be reasonable. Maybe we could get eslint to auto-fix that for us?

File: .eslintrc.js (hunk 1 above)
Comment (jcoyne) [reply .eslintrc.js]: yes, that's right.

File: src/components/editor/property/InputLang.jsx (hunk 2 above)
Comment (jcoyne) [reply src/components/editor/property/InputLang.jsx]: Yes, I was trying only to change the `__test__` imports in this PR, but I think we could change the imports in `src` too.

**Comment by jcoyne** [comment]
@jermnelson conflict resolved.

**Pull request merged**